autoload/lightline/colorscheme/Dracula.vim

```diff
@@ -0,0 +1,42 @@
+" =============================================================================
+" Filename: autoload/lightline/colorscheme/Dracula.vim
+" Author: adamalbrecht
+" License: MIT License
+" Last Change: 2018/04/11
+" =============================================================================
+
+let s:black    = g:dracula_palette.bg
+let s:gray     = g:dracula_palette.selection
+let s:white    = g:dracula_palette.fg
+let s:darkblue = g:dracula_palette.comment
+let s:cyan     = g:dracula_palette.cyan
+let s:green    = g:dracula_palette.green
+let s:orange   = g:dracula_palette.orange
+let s:purple   = g:dracula_palette.purple
+let s:red      = g:dracula_palette.red
+let s:yellow   = g:dracula_palette.yellow
+
+if exists('g:lightline')
+
+  let s:p = {'normal': {}, 'inactive': {}, 'insert': {}, 'replace': {}, 'visual': {}, 'tabline': {}}
+  let s:p.normal.left = [ [ s:black, s:purple ], [ s:cyan, s:gray ] ]
+  let s:p.normal.right = [ [ s:black, s:purple ], [ s:white, s:darkblue ] ]
+  let s:p.inactive.right = [ [ s:black, s:gray ], [ s:white, s:black ] ]
+  let s:p.inactive.left =  [ [ s:cyan, s:black ], [ s:white, s:black ] ]
+  let s:p.insert.left = [ [ s:black, s:green ], [ s:cyan, s:gray ] ]
+  let s:p.replace.left = [ [ s:black, s:red ], [ s:cyan, s:gray ] ]
+  let s:p.visual.left = [ [ s:black, s:orange ], [ s:cyan, s:gray ] ]
+  let s:p.normal.middle = [ [ s:white, s:gray ] ]
+  let s:p.inactive.middle = [ [ s:white, s:gray ] ]
+  let s:p.tabline.left = [ [ s:darkblue, s:gray ] ]
+  let s:p.tabline.tabsel = [ [ s:cyan, s:black ] ]
+  let s:p.tabline.middle = [ [ s:darkblue, s:gray ] ]
+  let s:p.tabline.right = copy(s:p.normal.right)
+  let s:p.normal.error = [ [ s:red, s:black ] ]
+  let s:p.normal.warning = [ [ s:yellow, s:black ] ]
+
+  let g:lightline#colorscheme#Dracula#palette = lightline#colorscheme#flatten(s:p)
```

Comment:
I won't pretend to know anything about lightline, although I spent some time browsing the repo/code and it looks nice. *Assuming* this works, LGTM, but I haven't experimented with it or its colors.

Reply:
Yeah, I had no experience with it either. 

I downloaded it and made sure it worked while after I wrote this in so I can confirm that it looks at least as good as it did before. And I can also confirm that placement of the lightline plugin with respect to this theme doesn't matter either when loading using (at least) vim-plug.